.gitlab-ci.yml

```diff
@@ -0,0 +1,102 @@
+variables:
+  DOCKER_REPOSITORY: "mozmeao/basket"
+  DOCKER_IMAGE_TAG: "${DOCKER_REPOSITORY}:${CI_COMMIT_SHORT_SHA}"
+  # Cannot use $DOCKER_IMAGE_TAG because it contains other variables itself.
+  DOCKER_RUN_CMD: "docker run --env-file .docker-env ${DOCKER_REPOSITORY}:${CI_COMMIT_SHORT_SHA}"
+
+stages:
+  - build
+  - test
+  - deploy
+
+build-images:
+  stage: build
+  tags:
+    - meao
+  script:
+    - docker build -t ${DOCKER_IMAGE_TAG} --build-arg GIT_SHA=${CI_COMMIT_SHA} --pull .
+    - docker push ${DOCKER_IMAGE_TAG}
+
+.test-stage:
+  stage: test
+  variables:
+    DOCKER_DEBUG: "False"
+    DOCKER_ALLOWED_HOSTS: "*"
+    DOCKER_SECRET_KEY: "foo"
+    DOCKER_DATABASE_URL: "sqlite:///"
+    DOCKER_SITE_URL: "http://localhost:8000"
+    DOCKER_CACHE_URL: "dummy://"
+    DOCKER_ENABLE_ADMIN: "True"
+    DOCKER_SECURE_SSL_REDIRECT: "False"
```

Comment:
should these match the values in [docker-compose.yaml](https://github.com/mozmeao/basket/blob/master/docker-compose.yml#L80-L86)?

Reply:
No, we don't want or need mysql & redis container dependencies for things like flake8